tests/src/drivers/test_driver_aead.c

```diff
@@ -15,13 +15,16 @@
 
 #include "mbedtls/constant_time.h"
 
-#if defined(MBEDTLS_TEST_LIBTESTDRIVER1)
+#if defined(TF_PSA_CRYPTO_TEST_LIBTESTDRIVER1)
+typedef psa_key_attributes_t libtestdriver1_psa_key_attributes_t;
```

Comment:
Since this is repeated, it feels like it should be in a header somewhere, probably `tests/include/test/drivers/test_driver_common.h`.